code/starfield/starfield.cpp

```diff
@@ -203,36 +184,41 @@ static int Mission_env_map = -1;
 static bool Env_cubemap_drawn = false;
 static bool Irr_cubemap_drawn = false;
 
-void stars_release_debris_vclips(debris_vclip *vclips)
+int get_motion_debris_by_name(SCP_string name) {
+	for (int i = 0; i < (int)Motion_debris_info.size(); i++) {
+		if (!stricmp(Motion_debris_info[i].name.c_str(), name.c_str())) {
```

Comment:
This can use `SCP_string_lcase_equal_to` since both sides are SCP_string

Reply:
k